prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1345,101 +1389,42 @@ periodics:
 
 # Experimental prowjob to verify the new create-clusters flow on standard.
 # If stable, this is intended to replace the above testgroup-based definitions
-- name: standard-regular
-  interval: 1h
+- <<: *config-sync-ci-job-v2
+  name: kpt-config-sync-standard-regular
   annotations:
     testgrid-dashboards: googleoss-kpt-config-sync-misc
     testgrid-tab-name: standard-regular
-  cluster: build-kpt-config-sync
-  decorate: true
-  decoration_config:
-    timeout: 2h
-  extra_refs:
-  - org: GoogleContainerTools
-    repo: kpt-config-sync
-    base_ref: main
   spec:
-    serviceAccountName: e2e-test-runner
+    <<: *config-sync-ci-job-spec-v2
     containers:
-    - image: us-docker.pkg.dev/kpt-config-sync-ci-artifacts/test-infra/gke-e2e:v1.0.0-590be013d
-      command:
-      - make
+    - <<: *config-sync-ci-container-v2
       args:
-      - test-e2e-gke-ci
-      env:
-      - name: GKE_E2E_TIMEOUT
-        value: 2h
-      - name: GCP_PROJECT
-        value: kpt-config-sync-ci-main
-      - name: GCP_NETWORK
-        value: prow-e2e-network-1
-      - name: GCP_SUBNETWORK
-        value: prow-e2e-subnetwork-1
-      - name: GCP_ZONE
-        value: us-central1-a
-      - name: GKE_RELEASE_CHANNEL
-        value: regular
-      - name: E2E_NUM_CLUSTERS
-        value: "10"
-      resources:
-        requests:
-          memory: "8Gi"
-          cpu: "4000m"
-    nodeSelector:
-      # This job requires 8vCPUs or less, so it is "small".
-      cloud.google.com/gke-nodepool: small-job-pool
+      - 'GKE_E2E_TIMEOUT=2h'
+      - 'GCP_SUBNETWORK=prow-e2e-subnetwork-1'
+      - 'GCP_ZONE=us-central1-a'
```

Comment:
we sure we want to use this zone?
is there another that might be less likely to have capacity outages and slow cluster creation?

Reply:
the zone we're using for standard isn't causing any major issues, it's the regional autopilot clusters that sometimes timeout.
See:
- autopilot: https://testgrid.k8s.io/googleoss-kpt-config-sync-misc#autopilot-regular
- standard: https://testgrid.k8s.io/googleoss-kpt-config-sync-misc#standard-regular

I expect using the lazily created clusters should resolve flakiness associated with cluster creation. I'm not familiar with data on which regions may have more capacity, I expect we would need to do some experimentation. However, even with a different region I expect there will always be times that we would experience capacity issues